planner/core/explain.go

```diff
@@ -103,7 +103,9 @@ func (p *PhysicalIndexScan) AccessObject() string {
 func (p *PhysicalIndexScan) OperatorInfo(normalized bool) string {
 	buffer := bytes.NewBufferString("")
 	if len(p.rangeInfo) > 0 {
-		fmt.Fprintf(buffer, "range: decided by %v, ", p.rangeInfo)
+		if !normalized {
+			fmt.Fprintf(buffer, "range: decided by %v, ", p.rangeInfo)
```

Comment:
Why we don't reach this line if it's normalized?

Reply:
Because `p.rangeInfo` maybe contain exact range information, such as:

```sql
IndexScan_16	... table:bmsql_order_line, index:PRIMARY(ol_w_id, ol_d_id, ol_o_id, ol_number), range:[1 7 2257,1 7 2257], keep order:false
```

compare to

```sql
IndexScan_23 ... table:bmsql_order_line, index:PRIMARY(ol_w_id, ol_d_id, ol_o_id, ol_number), range:[1 6 2463,1 6 2463], keep order:false
```